plugins/source/aws/codegen/recipes/s3.go

```diff
@@ -107,6 +108,27 @@ func S3Resources() []*Resource {
 					},
 				}...),
 		},
+		{
+			SubService:          "lens_configurations",
```

Comment:
`s3_storage_lens_configurations`? I think that's too long so I went for "lens*"

Reply:
`s3_storage_lens_configurations` seems closer to the underlying SDK - AWS calls it `StorageLensConfigurations` so I think we should keep it as similar as possible, as long as we don't exceed table name length limits (which for `s3_storage_lens_configurations` we don't).